packages/angular/ssr/src/routes/ng-routes.ts

```diff
@@ -583,13 +583,13 @@ export async function getRoutesFromAngularRouterConfig(
 
       if (serverConfigRouteTree) {
         for (const { route, presentInClientRouter } of serverConfigRouteTree.traverse()) {
-          if (presentInClientRouter || route === '**') {
+          if (presentInClientRouter || route.endsWith('/**')) {
```

Comment:
Question: I assume this still matches if just `**` is in the root list of routes? We automatically pre-pending a `/` right?

Reply:
Yes